pkg/install/deploystorage.go

```diff
@@ -256,47 +256,38 @@ func (i *Installer) deployStorageTemplate(ctx context.Context, installConfig *in
 		return err
 	}
 
-	var g graph
-
 	exists, err := i.graphExists(ctx)
-	if err != nil {
+	if err != nil || exists {
```

Comment:
If there was an error checking if the graph exists, or if it _does_ already exist, just return (don't save the graph).